src/seeds.escrow.cpp

```diff
@@ -14,15 +14,9 @@ void escrow::reset() {
     }
 }
 
-
-void escrow::init_balance(name user) {
-    auto it = sponsors.find(user.value);
-    
-}
-
 void escrow::check_asset(asset quantity) {
     check(quantity.is_valid(), "invalid asset");
-    check(quantity.symbol == seeds_symbol, "invalid asset");
+    check(quantity.symbol == utils::seeds_symbol, "invalid asset");
```

Comment:
Just coment - in other places I've added value > 0 because generally we always expect quantities to be positive - all of the contracts are written with positive quantities in mind. 

But in theory quantity can be negative, which made me think this may be an attack vector on some parts of our code, passing negative quantities. 

Let me know what you think - do we ever want to allow negative quantity? Because I don't think so!